Doc/library/inspect.rst

```diff
@@ -40,11 +40,28 @@ attributes:
 +-----------+-------------------+---------------------------+
 | Type      | Attribute         | Description               |
 +===========+===================+===========================+
-| module    | __doc__           | documentation string      |
+| module    | __name_           | name of the module        |
++-----------+-------------------+---------------------------+
+|           | __doc__           | documentation string      |
++-----------+-------------------+---------------------------+
+|           | __cached__        | the path where a compiled |
+|           |                   | version is/should be      |
```

Comment:
There isn't any "should be".  Either it's stored there or `__cached__` isn't set.

Reply:
This should go below `__file__`.